getting-started/trino/docker-compose-trino.yml

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+services:
+  polaris:
+    build:
+      context: .
```

Comment:
Does this context work in your environment? I had do change it to `../../` because Dockerfile exists in the project root directory.

Reply:
you're right, this should be `../../`, it was accidentally removed when i copied over the new setup in https://github.com/apache/polaris/pull/42/commits/9c2be7f6cb270ec47944987de652535f00a5350a

Reply:
updated! thanks